relayer/chains/icon/event_parse.go

```diff
@@ -82,3 +82,85 @@ func (p *Provider) parseCallMessage(e *types.EventLog, eventType string, height
 		Src:           src[0],
 	}, nil
 }
+
+// Parse Event
+func (p *Provider) parseMessageEvent(notifications *types.EventNotification) ([]*providerTypes.Message, error) {
+	height, err := notifications.Height.BigInt()
```

Comment:
A similar method already exists for getting event from event logs. Can you refactor them or reuse the existing method instead of creating almost duplicate method for the same purpose.

Reply:
lets see until next release.